src/components/WorkChart.astro

```diff
@@ -0,0 +1,17 @@
+---
+type Props = {
+  img: string;
+  chartModifier: string | null;
+  href: string | null;
+};
+
+const { img, chartModifier,href} = Astro.props;
+---
+
+<div class="work-chart">
+    <img src={img} alt="" class="work-chart__image">
+    <div class="work-chart__content">
+        <div class={`work-chart__content-text ${chartModifier}`}> <slot /> </div>
+        <a href={href}><img src="/assets/icons/right-arrow.png" class="work-chart__content-arrow"></a>
```

Comment:
Two main things here:
1. Please update the CSS classes to BEM you have it backawrds.
2. Change the right-arrow to be a pseudo-element instead of an img tag. I want a cleaner HTML here.